### DISCUSSION TITLE
Can I bind a DataGrid row to a collection/array?

### DISCUSSION THREAD

**Discussion by g-bortoletto** [AvaloniaUI/Avalonia#13561] [Q&A]
What I mean is, imagine you have a class like 

    public class Foo
    {
        public string[] headers;
        public string[] values;
    }

Would it be possible to bind the headers row from a DataGrid to the headers variable and the values to a row?

**Comment by timunie** [comment]
Maybe this Q&A https://github.com/AvaloniaUI/Avalonia/discussions/11785 has some hints for you.
